specification/ai/Azure.AI.Agents/tools/models.tsp

```diff
@@ -144,6 +144,30 @@ model ConnectedAgentToolDefinition extends ToolDefinition {
   connectedAgent: ConnectedAgentDetails;
 }
 
+@doc("The input definition information for a Browser Automation tool as used to configure an agent.")
+@added(Versions.v2025_05_15_preview)
+model BrowserAutomationToolDefinition extends ToolDefinition {
+  @doc("The object type, which is always 'browser_automation'.")
+  type: "browser_automation";
+
+  @doc("The browser_automation tool definition configuration.")
+  @maxItems(1)
+  @encodedName("application/json", "browser_automation")
```

Comment:
Isn't maxItems used for array? I don't think it's needed for strings. 

Same in other locations